File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        n_set = set(nums)
        mx = 1
        for n in nums:
            if (n - 1) not in n_set:
                cnt = 1
                t  = n + 1
                while t in n_set:
                   cnt += 1
                   t += 1
                mx = max(cnt, mx)
        return mx 
                   
        

File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> result(nums.size()*2);
        for (int i = 0; i < nums.size(); i++) {
            result[i + nums.size()] = result[i] = nums[i];
        }
        return result;
    }
};